app/controllers/api/v1/authors_controller.rb

```diff
@@ -0,0 +1,54 @@
+class Api::V1::AuthorsController < ApplicationController
+  before_action :load_author, only: %i(show update destroy)
+
+  def index
+    authors = Author.order_name
+    @authors = ActiveModelSerializers::SerializableResource.new(authors, {})
+    render json: {status: true, data: {authors: @authors}}, status: :ok
+  end
+
+  def show
+    @author = ActiveModelSerializers::SerializableResource.new(@author, {})
+    render json: {status: true, data: {author: @author}}, status: :ok
+  end
+
+  def create
+    @author = Author.new author_params
+    if @author.save
+      @author = ActiveModelSerializers::SerializableResource.new(@author, {})
+      render json: {status: true, data: {author: @author}}, status: :created
+    else
+      render json: {status: false, error: @author.errors}, status: :unprocessable_entity
+    end
+  end
+
+  def update
+    if @author.update author_params
+      @author = ActiveModelSerializers::SerializableResource.new(@author, {})
```

Comment:
```suggestion
      @author = AuthorSerializer.new(@author)
```
syntax này thử chạy ko e ?

Reply:
syntax này ko chạy a ơi, e thử rồi

Reply:
lỗi gì á, chụp a xem thử nha, test bằng console

Reply:
dạ e đã fix được rồi a

Reply:
bị gì á ?